nvflare/private/fed/app/utils.py

```diff
@@ -98,10 +98,14 @@ def create_admin_server(fl_server: FederatedServer, server_conf=None, args=None,
 
 
 def version_check():
-    if sys.version_info >= (3, 11):
-        raise RuntimeError("Python versions 3.11 and above are not yet supported. Please use Python 3.8, 3.9 or 3.10.")
+    if sys.version_info >= (3, 13):
```

Comment:
Is this all we need to do? We need to run all examples and CICD tests.

Reply:
I ran a couple of examples, our implementations are compatible with the Python 3.12. Yes, we need to run all the examples to certify.

Reply:
@chesterxgchen Last time is migration from 3.8 to 3.10. We did make the changes to solve the in-compatible issues. Those changes already included in the latest code bases.

Reply:
When installing the current 2.5.0rc8 on my environment I got [this protobuf version](https://pypi.org/project/protobuf/5.28.0/).

It supports Python 3.12 :tada: